client/src/component/ReviewList.jsx

```diff
@@ -11,7 +11,16 @@ class ReviewList extends React.Component {
     const { reviews } = this.props;
     return (
       <div>
-        {reviews.map(review => <Review review={review} key={review.message_id} />)}
+        {reviews.map((review) => {
+          // This will split the message in two
+          const splitMessage = review.message.split('');
+          const firstHalfOfMessage = splitMessage.slice(0, 600);
+          const secondHalfOfMessage = splitMessage.slice(600, splitMessage.length);
+          review.firstHalfOfMessage = firstHalfOfMessage.join('');
```

Comment:
this will mutate every object in the array. Maybe, that's not what we want to do.